verifier/src/error.rs

```diff
@@ -21,6 +21,8 @@ pub enum Error {
     TcbInfoNotYetValid,
     /// TCB info expired
     TcbInfoExpired,
+    /// TCB info version mismatch, expecting 3 got {0}
```

Comment:
Similar comment here regarding the hardcoded `3`

Reply:
Done in the most recent